tests/queries/0_stateless/02184_hash_functions_and_ip_types.reference

```diff
@@ -0,0 +1,43 @@
+Row 1:
+──────
+ipv4:                                        1.2.3.4
+wyHash64(toIPv4('1.2.3.4')):                 13096729196120951355
+xxHash32(toIPv4('1.2.3.4')):                 2430391091
+gccMurmurHash(toIPv4('1.2.3.4')):            5478801830569062645
+murmurHash2_32(toIPv4('1.2.3.4')):           1658978282
+javaHashUTF16LE(toIPv4('1.2.3.4')):          24190
+intHash64(toIPv4('1.2.3.4')):                5715546585361069049
+intHash32(toIPv4('1.2.3.4')):                3152671896
+metroHash64(toIPv4('1.2.3.4')):              5715546585361069049
+farmHash64(toIPv4('1.2.3.4')):               5715546585361069049
+hex(sipHash128(toIPv4('1.2.3.4'))):          DBB6A76B92B59789EFB42185DC32311D
+hex(murmurHash3_128(toIPv4('1.2.3.4'))):     549E9EF692591F6BB55874EF9A0DE88E
+jumpConsistentHash(toIPv4('1.2.3.4'), 42):   37
+sipHash64(toIPv4('1.2.3.4')):                10711397536826262068
+kostikConsistentHash(toIPv4('1.2.3.4'), 42): 0
```

Comment:
Cosmetic: Looking at #35553, `yandexConsistentHash` was renamed to `kostikConsistentHash`, you could include them side-by-side in the test (or simply omit `yandexConsistentHashing`).

EDIT: Somehow, `kostikConsistentHash` isn't documented [here](https://clickhouse.com/docs/en/sql-reference/functions/hash-functions/). Perhaps you could at least include a mention of it, see "Functions/kostikConsistentHash.cpp".